fission-web-server/README.md

```diff
@@ -41,3 +41,49 @@ $ psql
 ```
 
 Migrations will be performed automatically when running the server
+
+### Using Docker
+
+To mimic the full "fission stack" for local development, you can use the included `docker-compose.yml`. This assumes that you have Docker installed locally:
+
+1. Copy the env.example file to `.env` (these values are used by docker compose). Edit as desired (defaults will work)
+2. Start docker compose: `docker compose up -d`. This will start postgres () and powerdns containers, including a local DNS resolver.
+3. Create the necessary zones (specified in `.env`) e.g.: `docker compose exec dns-auth pdnsutil create-zone fissionapp.test`
+4. Point your local DNS resolver to localhost (on macOS this is under System Preferences > Network > Advanced).
```

Comment:
I'm doing something wrong at this step 😅 

<img width="626" alt="Screen Shot 2022-02-24 at 15 58 58" src="https://user-images.githubusercontent.com/1052016/155627554-bc6f1696-ae67-4516-bdbf-f45e28e61300.png">

<img width="492" alt="Screen Shot 2022-02-24 at 15 59 31" src="https://user-images.githubusercontent.com/1052016/155627624-d50f0330-6b3a-4c02-847f-cda603f27858.png">
<img width="502" alt="Screen Shot 2022-02-24 at 15 59 36" src="https://user-images.githubusercontent.com/1052016/155627625-4de3e3b8-c9a3-4932-8f7a-f442e8d8a89e.png">

Reply:
(I _did_ hit `Apply` in the main Networking menu, so it's not that)

Reply:
Getting a funny response from dig though:
```sh
[nix-shell:~/work/fission/fission]$ dig fissionapp.test

; <<>> DiG 9.16.23 <<>> fissionapp.test
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 2092
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;fissionapp.test.		IN	A

;; AUTHORITY SECTION:
fissionapp.test.	3600	IN	SOA	a.misconfigured.dns.server.invalid. hostmaster.fissionapp.test. 0 10800 3600 604800 3600

;; Query time: 56 msec
;; SERVER: 127.0.0.1#53(127.0.0.1)
;; WHEN: Fri Feb 25 12:17:24 CET 2022
;; MSG SIZE  rcvd: 125
```

Reply:
> a.misconfigured.dns.server.invalid

:thinking:

Reply:
> I'm doing something wrong at this step 😅

I've noticed that, for example, Tailscale DNS will override the setting here. If you disconnect Tailscale (or other VPNs) does that help? 

But, I'm beefing up the README anyway because I clearly skipped some details

Reply:
> > a.misconfigured.dns.server.invalid

ah. harmless - https://docs.powerdns.com/authoritative/settings.html#default-soa-name

I don't fully understand why they do this, but it will still work anyway.

Reply:
I've updated the docs to be a bit more explicit and included a troubleshooting section.

Have another look and see if that makes more sense? 🙏